contracts/AnchorVault.vy

```diff
@@ -275,229 +186,21 @@ def change_admin(new_admin: address):
-    _liquidations_admin: address,
-    _no_liquidation_interval: uint256,
-    _restricted_liquidation_interval: uint256
-):
-    assert _restricted_liquidation_interval >= _no_liquidation_interval
-
-    self.liquidations_admin = _liquidations_admin
-    self.no_liquidation_interval = _no_liquidation_interval
-    self.restricted_liquidation_interval = _restricted_liquidation_interval
-
-    log LiquidationConfigUpdated(
-        _liquidations_admin,
-        _no_liquidation_interval,
-        _restricted_liquidation_interval
-    )
-
-
-@external
-def set_liquidation_config(
-    _liquidations_admin: address,
-    _no_liquidation_interval: uint256,
-    _restricted_liquidation_interval: uint256,
-):
-    """
-    @dev Sets the liquidation config consisting of liquidation admin, the address that is allowed
-         to sell stETH rewards to UST during after the no-liquidation interval ends and before
-         the restricted liquidation interval ends, as well as both intervals.
-
-    Can only be called by the current admin address.
-    """
-    self._assert_admin(msg.sender)
-    self._set_liquidation_config(
-        _liquidations_admin,
-        _no_liquidation_interval,
-        _restricted_liquidation_interval
-    )
-
-
-@internal
-def _set_anchor_rewards_distributor(_anchor_rewards_distributor: bytes32):
-    self.anchor_rewards_distributor = _anchor_rewards_distributor
-    log AnchorRewardsDistributorUpdated(_anchor_rewards_distributor)
-
-
-@external
-def set_anchor_rewards_distributor(_anchor_rewards_distributor: bytes32):
-    """
-    @dev Sets the Terra-side UST rewards distributor contract allowing Terra-side bETH holders
-         to claim their staking rewards in the UST form.
-
-    Can only be called by the current admin address.
-    """
-    self._assert_admin(msg.sender)
-    self._set_anchor_rewards_distributor(_anchor_rewards_distributor)
-
-
-@external
-def configure(
-    _bridge_connector: address,
-    _rewards_liquidator: address,
-    _insurance_connector: address,
-    _liquidations_admin: address,
-    _no_liquidation_interval: uint256,
-    _restricted_liquidation_interval: uint256,
-    _anchor_rewards_distributor: bytes32,
-):
-    """
-    @dev A shortcut function for setting all admin-configurable settings at once.
-
-    Can only be called by the current admin address.
-    """
-    self._assert_admin(msg.sender)
-    self._set_bridge_connector(_bridge_connector)
-    self._set_rewards_liquidator(_rewards_liquidator)
-    self._set_insurance_connector(_insurance_connector)
-    self._set_liquidation_config(
-        _liquidations_admin,
-        _no_liquidation_interval,
-        _restricted_liquidation_interval
-    )
-    self._set_anchor_rewards_distributor(_anchor_rewards_distributor)
-
+    raise "Set emergency admin is disabled"
 
 @internal
 @view
-def _get_rate(_is_withdraw_rate: bool) -> uint256:
+def _get_rate() -> uint256:
     steth_balance: uint256 = ERC20(self.steth_token).balanceOf(self)
```

Comment:
may depart from `balanceOf` → `sharesOf`

Reply:
leave the `balanceOf` approach